pom.xml

```diff
@@ -429,7 +462,13 @@
                     <source>11</source>
                     <target>11</target>
                     <encoding>UTF-8</encoding>
-
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
+                            <version>${lombok.version}</version>
+                        </path>
+                    </annotationProcessorPaths>
```

Comment:
Should all these dependencies be added? Is this a mistake of cherry-picking from the master server-web? Kindly check

Reply:
will check

Reply:
Kindly check this again